testsuite/MDAnalysisTests/coordinates/test_netcdf.py

```diff
@@ -299,6 +300,23 @@ def test_box(self, universe, index, expected):
         assert_almost_equal(self.box_refs[expected], universe.dimensions)
 
 
+class TestNCDFReader4(object):
+    """NCDF Trajectory exported by cpptaj, without `time` variable.
+    """
+    prec = 3
+
+    @pytest.fixture(scope='class')
+    def u(self):
+        return mda.Universe(CPPTRAJ_TRAJ_TOP,
+                            [CPPTRAJ_TRAJ, CPPTRAJ_TRAJ])
+
+    def test_positions_1(self, u):
+        """Check positions on first frame"""
+        ref_times = [0.0, 1.0, 2.0, 3.0, 4.0, 5.0]
+        time_list = [ts.time for ts in u.trajectory]
+        assert ref_times == time_list
+
```

Comment:
Because we area also adding that extra exception on `dt`, could you also add a quick check that calling dt on a random frame gives you an expected result? (technically it's done implicitly by calling times, but it would be good to have direct coverage of its behaviour)

Reply:
Done, I used assert_equal as I saw it used in the tests above, doesn't a simple ‘==‘ comparison work in this case?

Reply:
For single floats assert_equal is indeed not worth it (last I checked it's nearly an order of magnitude slower!).

In this particular case it's unlikely that floats are going to be a problem, so a simple assert would be fine. Otherwise the way to do it (when you have a chance of float precision issues), is to use pytest.approx like so: https://github.com/MDAnalysis/mdanalysis/blob/a05a5245042361fd4fa27f03993f34cee2cda901/testsuite/MDAnalysisTests/analysis/test_align.py#L569